src/commands/org/resume/scratch.ts

```diff
@@ -30,6 +30,7 @@ export default class EnvResumeScratch extends SfCommand<ScratchCreateResponse> {
   public static readonly flags = {
     'job-id': Flags.salesforceId({
       char: 'i',
+      length: 'both',
```

Comment:
I'm ok with it as long as it's been tested (automated or manual) to work with both.